pysteps/postprocessing/probmatching.py

```diff
@@ -259,6 +260,49 @@ def _get_error(scale):
     return shift, scale, R.reshape(shape)
 
 
+def resample_distributions(extrapolation_cascade, model_cascade, weight_extrapolation):
+    """
+    Merges two distributions (e.g. from the extrapolation nowcast and NWP in the blending module)
+    to effectively combine two distributions for the probability matching without losing extremes.
+    Parameters
+    ----------
+    extrapolation_cascade: array_like
+        One of the two arrays from which the distribution should be samples (e.g. the extrapolation
+        cascade).
+    model_cascade: array_like
+        One of the two arrays from which the distribution should be samples (e.g. the NWP (model)
+        cascade).
+    weight_extrapolation: float
```

Comment:
we should make clear that this specifies the probability of sampling the first array

```suggestion
    probability_first_array: float
```

Reply:
Thanks! Added it in the commit below :)